scene/3d/ray_cast_3d.cpp

```diff
@@ -32,6 +32,7 @@
 
 #include "collision_object_3d.h"
 #include "mesh_instance_3d.h"
+#include "modules/gridmap/grid_map.h"
```

Comment:
Modules can be disabled, so this will break the build for `module_gridmap_enabled=no`. So it needs checks:
```suggestion

#include "modules/modules_enabled.gen.h" # For gridmap.
#ifdef MODULE_GRIDMAP_ENABLED
#include "modules/gridmap/grid_map.h"
#endif
```
And same `#ifdef`s where `GridMap` is actually used.

The added dependency on a module somewhat breaks encapsulation, but it's not like it's without precedent in the engine so if there's no way to do this without special casing `GridMap` here, I guess we'll have to live with it.